Code Review: Issue 93: Bug Fix for joi validation

mrsauravsahu opened the pull request:
#93 
I have removed the POST /api/user/loan API, as it will be implemented in #17 separately.

mrsauravsahu committed:
add runInBand option to jest in the scripts section of package.json

add runInBand option to jest in the scripts section of package.json

mrsauravsahu committed:
add seeder to add facebookUserId to facebooks table, for testing

add seeder to add facebookUserId to facebooks table, for testing

mrsauravsahu committed:
Fix tests: replace userId with access token. For tests, pass accesstoken using environment variable.

Fix tests: replace userId with access token. For tests, pass accesstoken using environment variable.

mrsauravsahu committed:
Fix joi validation.

Fix joi validation.

Ghost---Shadow commented:
Buddy says `ERROR: Unable to find migration: 20180226163640-create-twitters.js`

Did you rebase?

mrsauravsahu committed:
add facebook user seed data for testing.

add facebook user seed data for testing.

mrsauravsahu committed:
Update test to use correct userId

Update test to use correct userId

mrsauravsahu commented:
@Ghost---Shadow It was because all builds from `issue*` branch were running in the same pipepline, and migrations wouldn't undo properly. Updated the pipeline to try to drop and recreate the database before every build now. And also added Shubham's access token in the build as an environment variable.
